mkbuildimg

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+cat Dockerfile files/Dockerfile_build > Dockerfile_tmp
```

Comment:
Please add Dockerfile_tmp to .gitignore, so we don't end up committing it by mistake.